2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,39 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return (this.volume = 100);
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    if (this.volume === 100) {
+      console.log(`You cannot exceed the bottle capacity`);
+    } else {
+      return (this.volume += 10);
+    }
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    if (this.volume === 0) {
+      console.log(`You cannot drink more than its actual contents`);
```

Comment:
Really great job logging out a message to indicate something went wrong/didn't happen 👍